src/utils/fetch.util.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ *
+ * @description Do not use this for nested objects or array. It is only for simple objects.
+ */
+export const convertObjectToQueryParams = (object: { [key: string]: number | string }) => {
```

Comment:
Instead of doing this, we can use inbuild [URL object ](https://developer.mozilla.org/en-US/docs/Web/API/URL/searchParams). to add query parameters.

Here is an example

```javascript 
const getApiWithLanguageParams = (url) => {
    let apiUrl = new URL(makeApiURl(url));
    const appLanguage = localStorage.getItem('appLanguage');
    if (appLanguage) {
        apiUrl.searchParams.set('lng', appLanguage)
    } else {
        apiUrl.searchParams.set('lng', '1');
    }
    return apiUrl.href;
}
````